CMakeLists.txt

```diff
@@ -897,10 +897,8 @@ if(NOT MSVC)
   append_cxx_flag_if_supported("-Wno-error=inconsistent-missing-destructor-override" CMAKE_CXX_FLAGS)
   append_cxx_flag_if_supported("-Wconstant-conversion" CMAKE_CXX_FLAGS)
   append_cxx_flag_if_supported("-Wno-invalid-partial-specialization" CMAKE_CXX_FLAGS)
-  append_cxx_flag_if_supported("-Wno-unused-private-field" CMAKE_CXX_FLAGS)
```

Comment:
Why are these removed from here?

Reply:
To identify and remove unused private field in torch. I think it a useful feature and if there is a third-party library requiring this fix, it should be added to its CMake target rather than enabled globally.

Reply:
I think I'm just confused about the use of -W flag here. Doesn't removing this line actually removes the warning for the top level library?

Reply:
Removing it should apply to the whole library.